autolux/light.py

```diff
@@ -0,0 +1,12 @@
+# A small compatibility layer for `light`
+# https://github.com/haikarainen/light
+
+import opts
+from run_cmd import run_cmd
+
+def set_brightness(new_level, time):
+    run_cmd("light -apS {:f}".format(new_level))
```

Comment:
should this just be "light -S {:f}"? 

I installed light from the github repo and when I run this branch locally, I get:

```
light: invalid option -- 'a'
light: invalid option -- 'p'
```

is '-ap' supported by your instance of light? When I run "light -apS 10", it still works but I do get the above output

Reply:
Weird. Since you are using the latest master, I guess it means they have removed these two options. They were the default ones anyway, so we can safely use `light -S` and `light -G` instead.